api/python/quilt3/packages.py

```diff
@@ -852,7 +850,7 @@ def _set_commit_message(self, msg):
 
 
     @ApiTelemetry("package.build")
-    def build(self, name=None, registry=None, message=None):
```

Comment:
This could break user code, right?

Reply:
Not really: in the old code, `name=None` would just cause an exception, so it was already broken.